addons/binding/org.openhab.binding.miele/ESH-INF/thing/xgw3000.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="miele"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Miele Bridge -->
+    <bridge-type id="xgw3000">
+        <label>Miele Bridge</label>
+        <description>The miele bridge represents the Miele@Home XGW3000 gateway.</description>
+
+        <properties>
+        	<property name="vendor">Miele</property>
+        </properties>
+        
+        <config-description>
+            <parameter name="ipAddress" type="text">
+                <context>network_address</context>
+                <label>Network Address</label>
+                <description>Network address of the Miele@Home gateway.</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="interface" type="text">
+                <context>network_address</context>
+                <label>Network Address of the Multicast interface</label>
+                <description>Network address of openHAB host interface where the bindign will listen for multicast events coming from the Miele@Home gateway</description>
+                <required>true</required>
```

Comment:
>  I am not sure if the "first network interface" found is really the one a user is expecting to be found.

Well, he either specifies this (as he has to do right now) and if he doesn't, he shouldn't expect anything. But for all users with a single interface, all would be fine then.